packages/core/lib/console.js

```diff
@@ -161,18 +161,29 @@ class Console extends EventEmitter {
     // errors/warnings in child process - specifically the error re: having
     // multiple config files
     const spawnOptions = { stdio: ["inherit", "inherit", "pipe"] };
+    let spawnInput;
+    if (options.config) {
+      spawnInput =
+        "--config " +
+        options.config +
+        " --network " +
+        options.network +
+        " -- " +
+        inputStrings;
+    } else {
+      spawnInput = "--network " + options.network + " -- " + inputStrings;
+    }
```

Comment:
What about cleaning this up to be something like:

```suggestion
    const settings = ["config", "network"]
      .filter(setting => options[setting])
      .map(setting => `--${setting}=${options[setting]}`);
    const spawnInput = `${settings} -- ${inputStrings}`;
```

Reply:
This won't work, the spawnInput is incorrect. But I'd love to clean it up this way, will push a change in a couple of hours!

Reply:
Oh oops, forgot a join! Edited.

Reply:
Oops! Sorry I just pushed this without realizing you'd done it here! Same solution, adding the join. :) I also removed the `=` because it didn't seem necessary.

Reply:
It's not necessary, but using `=` seems to me like best practice; whenever writing examples, I always include it (there's no risk of misinterpreting what's a key/value pair vs. a boolean flag + separate positional arg)

Not worth changing now, though.